### DISCUSSION TITLE
climatology dictionary

### DISCUSSION THREAD

**Discussion by sherrieF** [dtcenter/METplus#1316] [Statistical Computation] [answered]
Hi,

   I am unable to get the anomanly correlations to work again.
In the GridStat.config file I have 
GRID_STAT_CLIMO_MEAN_FIELD = {name="UGRD"; level="P850";},{name="UGRD"; level="P500";}, {name="UGRD"; level="P250";}
on line 153.  The error I get is 
ERROR  : 
ERROR  : DictionaryEntry::dict_value() -> bad type
ERROR  : 
I ran this a while ago and it worked just fine.   When I comment out line 153 METplus runs just fine.
I am working on Hera.  My configuration file is /home/Sherrie.Fredrick/METplus/gridstat/GridStat.conf.C96_replay_winter 
The log file is /scratch2/BMC/gsienkf/Sherrie.Fredrick/C96_replay_winter/metout/logs/metplus.log.20211215164503

Many Thanks for your help.
Sherrie

**Comment by jprestop** [comment, marked as answer]
Hi @sherrieF. 

Per #1242 you can see in this [reply](https://github.com/dtcenter/METplus/discussions/1242#discussioncomment-1609147),  it appears that you need to make sure that you've run "module load metplus/4.1.0-beta3".  Can you please confirm that you are using that version on Hera?

**Comment by sherrieF** [comment]
Hi Julie,
    I was not running module load metplus/4.1.0-beta3.  I switched to this
module and everything ran just fine.
Many Thanks,
Sherrie

**Reply by jprestop** [reply]
Great!  Thanks for letting us know.  At this point I'm going to LOCK this conversation to prevent future posts. Our team has decided that once discussions have been answered, and the answer has been confirmed, we'll lock them. We want to encourage users to ask new questions in new discussions rather than posting to old ones. Hopefully that'll make the questions/answers easier for other users to follow.  

So if/when more issues/questions arise, please feel free to start a new discussion.
